src/status_im/ui2/screens/chat/messages/deleted_message.cljs

```diff
@@ -0,0 +1,19 @@
+(ns status-im.ui2.screens.chat.messages.deleted-message
```

Comment:
this code should not be here. 
Can you please move this to status_im2. We want to avoid adding any new code to status_im.

Reply:
got it, will fix

Reply:
done